app/controllers/miq_ae_class_controller.rb

```diff
@@ -1073,7 +1095,7 @@ def method_form_fields
   def update
     assert_privileges("miq_ae_class_edit")
     return unless load_edit("aeclass_edit__#{params[:id]}", "replace_cell__explorer")
-
+    
```

Comment:
Avoid trailing whitespaces.

(I'm surprised the bot didn't comment here to run rubocop)